app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -46,6 +47,13 @@ def redirect_if_blank_phone
       redirect_to new_user_session_path
     end
 
+    def track_mfa_added
+      mfa_user = MfaContext.new(current_user)
+      mfa_count = mfa_user.enabled_mfa_methods_count
+      analytics.multi_factor_auth_added_phone(enabled_mfa_methods_count: mfa_count)
```

Comment:
as long as we're here, WDYT about inlining some of this?
```suggestion
      analytics.multi_factor_auth_added_phone(
        enabled_mfa_methods_count: MfaContext.new(current_user).enabled_mfa_methods_count,
      )
```